elastic/kibana
# [TableListView] Add sorting dropdown

This PR adds a selectable dropdown to change the `<TableListView />` sort column. The select is controlled by the global state which alllows the sort to changed either by clicking the table column directly or by using the select.

<img width="1286" alt="Screenshot 2022-10-11 at 16 41 06" src="https://user-images.githubusercontent.com/2854616/195137933-8e74fddc-ed40-4668-87a3-67ec14e259e6.png">

## How to test

* Navigate to the Dashboard or visualisation app
* Change the table sort column by clicking on the header
* Verify that the select value has changed
* Change the select option and verify that the table column sort is updated

Fixes https://github.com/elastic/kibana/issues/135195
